spec/plugin_spec.rb

```diff
@@ -291,9 +284,9 @@ def api_get(url)
       end
 
       it 'conflicts on an existing static adress' do
-        data = api_post '/IpamDriver.RequestAddress', { 'PoolID' => 'test', 'Address' => '10.80.0.1'}
+        data = api_post '/IpamDriver.RequestAddress', { 'PoolID' => 'test', 'Address' => '10.80.0.1'},
+          expect_status: 400
```

Comment:
This isn't really a REST API, so do we need HTTP semantic error responses? :)

Needs a separate issue: #70